### ISSUE TITLE
False positive of DL3014 when -qqy option is supplied

### ISSUE THREAD

**Issue by timbru31** [hadolint/hadolint#131]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

DL3014 enforces to pass the --assume-yes option to apt-get, but when using it together with the quiet option, it produces a false positive (IMHO).

### Actual behavior

DL3014 should not be marked as an error when apt-get is e.g. called via -qy or -qqy (or vice versa)

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter UNKNOWN
```

This is weird. Just installed hadolint via Homebrew and got
```bash
==> Pouring hadolint-1.2.3.high_sierra.bottle.tar.gz
```

**Dockerfile** (trimmed down)

```Dockerfile
FROM openjdk:8-jre

RUN apt-get update -qqy && apt-get -qqy install \
    nodejs \
```

**Additional environment details (OS, stack version, etc.)**
* macOS High Sierra 10.13.2 (17C88)
* Homebrew 1.4.0 (Homebrew/homebrew-core (git revision dbce; last commit 2017-12-11))

**Comment by zemanlx** [comment]
@timbru31 we do not support using multiple options behind one `-`. If you split them to `-q -y` it should work. 
You can find accepted options in `apt-get install` [here](https://github.com/hadolint/hadolint/blob/master/src/Hadolint/Rules.hs#L216).

Maybe it can be useful to add `-qq` as an alternative to `-y`. But it looks like that it has its downsides. Why do you `-qq` in builds?

from `man apt-get`
```
-q, --quiet
           Quiet; produces output suitable for logging, omitting progress indicators. More
           q's will produce more quiet up to a maximum of 2. You can also use -q=# to set
           the quiet level, overriding the configuration file. Note that quiet level 2 
           implies -y; you should never use -qq without a no-action modifier such as -d,
           --print-uris or -s as APT may decide to do something you did not expect. 
           Configuration Item: quiet.
```

**Comment by timbru31** [comment]
Hi @zemanlx ,

I'm aware that I can split up the options into multiple, but wanted to report that combining them is producing a false positive.

I guess we use `-qq` to have a smaller log file, but honestly I can't remember exactly why.  
Thanks for the hint that it implies `-y`, too.
